NEWS.md

```diff
@@ -1,5 +1,7 @@
 # ggplot2 2.2.1.9000
 
+* Uses `rel()` more consistently in theme defaults (@baptiste).
```

Comment:
I think you can be a bit more explicit: use `rel()` to set line widths in theme defaults.